_includes/scripts.html

```diff
@@ -8,3 +8,7 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
+{% if layout.includes_touchpoints and page.category != "verify-your-identity" %}
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
+  <script src="{{'/assets/js/touchpoints_translations.js' | relative_url}}"></script>
```

Comment:
This is loading the source file, which probably probably works as-is everywhere currently, but since we have the expectation of being able to write modern JavaScript in those files or use `import` statements, it might not always work (especially in Internet Explorer).

Instead, I think we just need to point this to the built version, [like we do with `main.js`](https://github.com/18F/identity-site/blob/8aa25a05cebf1e298825c0512bae6c56eeb39fd6/_config.yml#L37):

```suggestion
  <script src="{{'/assets/js/build/touchpoints_translations.js' | relative_url}}"></script>
```